.github/workflows/ci.yml

```diff
@@ -354,23 +215,11 @@ jobs:
       PARACHAIN_DOCKER_IMAGE: ${{ github.event.inputs.parachainDocker || format('mangatasolutions/mangata-node:rococo-{0}', needs.init.outputs.GLOBAL_VERSION) }}
     steps:
       ####IDK, but this is neccesary for reports
-      -
-        name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      -
-        name: Download artifact
+      - name: Download node Docker image
         if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-build') && github.event.inputs.skipBuild != 'true' }}
-        uses: actions/download-artifact@v2
-        with:
-          name: rococo-docker-${{ needs.init.outputs.GLOBAL_VERSION }}
-          path: /tmp
-      -
-        name: Load Docker image
-        if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-build') && github.event.inputs.skipBuild != 'true' }}
-        run: |
-          docker load --input /tmp/rococo-docker-${{ needs.init.outputs.GLOBAL_VERSION }}.tar
-          docker image ls -a
+        run: docker pull ${{ env.PARACHAIN_DOCKER_IMAGE }}
```

Comment:
why this? are not we using `"mangatasolutions/mangata-node:rococo-${{ needs.init.outputs.GLOBAL_VERSION }}"` ?